docs/DeveloperGuide.md

```diff
@@ -664,7 +714,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * *a. At any time, User requests to [view help UC01](#use-case-uc01---view-help)
 
 
-#### Use case: UC11 - List bookings
+#### Use case: UC12 - List bookings
```

Comment:
Can update the link to this use case? cause previously was [list bookings UC11](#use-case-uc11---list-bookings) now need to change to [list bookings UC12](#use-case-uc12---list-bookings)